requirements_docs.txt

```diff
@@ -2,7 +2,7 @@ pywps>=4.5.1
 sphinx>=4.0
 nbsphinx
 ipython
-xclim==0.38
+xclim==0.43
 pint<0.20  # xclim < 0.39 breaks with pint 0.20
```

Comment:
Ditto.

Reply:
@aulemahal indicated that xclim is always pinned in finch? 
Also if I wait for #272 then integrate changes will the tests fix themselves?

Reply:
I'm talking specifically the `pint` pin, sorry!

Reply:
Merging should fix failures related to h5netcdf and the subset notebook, but not the ones from the "*_series" test helpers, these come from a change posterior to 0.40 (the xclim in 272).